tutor/plugins/openedx.py

```diff
@@ -111,6 +111,8 @@ def _mount_edx_platform_compose(
         ("openedx", "edx-ora2"),
         ("openedx", "edx-search"),
         ("openedx", "openedx-learning"),
+        ("openedx", "openedx-events"),
+        ("openedx", "opaque-keys"),
```

Comment:
What if instead of hard-coding, we add the regex, such as openedx-*? The same has been done for platform-plugin.

Reply:
I can't re-call anything right now. I think this should be good to go. 
@ahmed-arb What do you think?

Reply:
No it still only mounts repos that you explicitly request with `tutor mounts add`. This just tells tutor to make them available to all the `openedx` images: `lms`, `cms`, `lms-worker`, `cms-worker`, etc. and to install them when building python deps for those images.